cmake/modules/FindWavPack.cmake

```diff
@@ -0,0 +1,84 @@
+# This file is part of Mixxx, Digital DJ'ing software.
+# Copyright (C) 2001-2019 Mixxx Development Team
+# Distributed under the GNU General Public Licence (GPL) version 2 or any later
+# later version. See the LICENSE file for details.
+
+#[=======================================================================[.rst:
+FindWavPack
+-----------
+
+Finds the WavPack library.
+
+Imported Targets
+^^^^^^^^^^^^^^^^
+
+This module provides the following imported targets, if found:
+
+``WavPack::WavPack``
+  The WavPack library
+
+Result Variables
+^^^^^^^^^^^^^^^^
+
+This will define the following variables:
+
+``WavPack_FOUND``
+  True if the system has the WavPack library.
+``WavPack_INCLUDE_DIRS``
+  Include directories needed to use WavPack.
+``WavPack_LIBRARIES``
+  Libraries needed to link to WavPack.
+``WavPack_DEFINITIONS``
+  Compile defitions needed to use WavPack.
+
+Cache Variables
+^^^^^^^^^^^^^^^
+
+The following cache variables may also be set:
+
+``WavPack_INCLUDE_DIR``
+  The directory containing ``wavpack/wavpack.h``.
+``WavPack_LIBRARY``
+  The path to the WavPack library.
+
+#]=======================================================================]
+
+find_package(PkgConfig)
+pkg_check_modules(PC_WavPack QUIET wavpack)
```

Comment:
Why do we mix her camel case and capital case?

Reply:
Have a look at the cmake documentation regarding "Find Modules", which should answer most of your questions:
https://cmake.org/cmake/help/latest/manual/cmake-developer.7.html#id4
AFAIK this is the proper way(TM) to do it-

Reply:
Ah Ok, thank you for the link.